packages/lesswrong/components/dropdowns/comments/DeleteCommentDropdownItem.tsx

```diff
@@ -29,21 +30,27 @@ const DeleteCommentDropdownItem = ({comment, post, tag}: {
     });
   }
 
-  const handleUndoDelete = (event: React.MouseEvent) => {
+  const handleUndoDelete = async (event: React.MouseEvent) => {
     event.preventDefault();
-    void moderateCommentMutation({
-      commentId: comment._id,
-      deleted:false,
-      deletedReason:"",
-    }).then(() => flash({
-      messageString: "Successfully restored comment",
-      type: "success",
-    })).catch(/* error */);
+    try {
+      await moderateCommentMutation({
+        commentId: comment._id,
+        deleted:false,
+        deletedReason:"",
+      })
+      flash({
+        messageString: "Successfully restored comment",
+        type: "success",
+      });
+    } catch(e) {
+      flash(e.message);
```

Comment:
Just checking, `flash` accepts both objects and strings?

Reply:
Yep. The `type` thing is for optional styling (which came from Vulcan and I'm not sure it has an effect anymore).